datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/browse/BrowseResolver.java

```diff
@@ -0,0 +1,58 @@
+package com.linkedin.datahub.graphql.resolvers.browse;
+
+import com.linkedin.datahub.graphql.types.BrowsableEntityType;
+import com.linkedin.datahub.graphql.generated.BrowseInput;
+import com.linkedin.datahub.graphql.generated.BrowseResults;
+import com.linkedin.datahub.graphql.generated.EntityType;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+
+import static com.linkedin.datahub.graphql.resolvers.ResolverUtils.bindArgument;
+
+public class BrowseResolver implements DataFetcher<CompletableFuture<BrowseResults>> {
+
+    private static final int DEFAULT_START = 0;
+    private static final int DEFAULT_COUNT = 10;
```

Comment:
is this just page size?

Reply:
Default size, yep!

Reply:
(On the server we call these parameters "start" and "count". Resolving.